### PR TITLE
Added comments and fixed code formatting.

### PR DESCRIPTION
**Pull request by laum18**
Added regions and comments to methods and changed some variable names to follow coding standards.

### CODE REVIEW

File: EPPlus/Table/PivotTable/ExcelPivotCacheDefinition.cs [hunk 1]
```diff
@@ -49,182 +50,109 @@ public enum eSourceType
...
+		/// <summary>
+		/// The path of the data source worksheet.
+		/// </summary>
+		internal const string SourceWorksheetPath = "d:cacheSource/d:worksheetSource/@sheet";
+		
+		/// <summary>
+		/// The path of the data source cell range.
+		/// </summary>
+		internal const string SourceAddressPath = "d:cacheSource/d:worksheetSource/@ref";
+		private const string SourceNamePath = "d:cacheSource/d:worksheetSource/@name";
```
Comment (shawncrabtree) [inline EPPlus/Table/PivotTable/ExcelPivotCacheDefinition.cs]: Newline between these properties.

File: EPPlus/Table/PivotTable/ExcelPivotTableCollection.cs [hunk 2]
```diff
@@ -39,125 +39,143 @@ namespace OfficeOpenXml.Table.PivotTable
...
 	public class ExcelPivotTableCollection : IEnumerable<ExcelPivotTable>
 	{
-		List<ExcelPivotTable> _pivotTables = new List<ExcelPivotTable>();
-		internal Dictionary<string, int> _pivotTableNames = new Dictionary<string, int>();
-		ExcelWorksheet _ws;
+		#region Class Variables
+		/// <summary>
+		/// A list of pivot tables.
+		/// </summary>
+		internal Dictionary<string, int> myPivotTableNames = new Dictionary<string, int>();
```
Comment (EvanSchallerer) [inline EPPlus/Table/PivotTable/ExcelPivotTableCollection.cs]: Can we make the internal class variables private and just make whatever is accessing them outside of the class use the property instead?

File: EPPlus/Table/PivotTable/ExcelPivotTableCollection.cs [hunk 3]
```diff
@@ -39,125 +39,143 @@ namespace OfficeOpenXml.Table.PivotTable
...
+			{
+				return myPivotTables.Count;
+			}
+		}
+		
+		/// <summary>
+		/// Gets the pivot table Index starting at base 0.
+		/// </summary>
+		/// <param name="Index">The position of the pivot table.</param>
+		/// <returns></returns>
```
Comment (EvanSchallerer) [inline EPPlus/Table/PivotTable/ExcelPivotTableCollection.cs]: <returns> comment

File: EPPlus/Table/PivotTable/ExcelPivotTableFieldCollection.cs [hunk 4]
```diff
@@ -36,235 +36,304 @@
...
+		/// Gets the enumerator of the list.
+		/// </summary>
+		/// <returns>The enumerator.</returns>
 		public IEnumerator<T> GetEnumerator()
 		{
-			return _list.GetEnumerator();
+			return myList.GetEnumerator();
 		}
 
 		System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator()
```
Comment (zpfaltersack) [inline EPPlus/Table/PivotTable/ExcelPivotTableFieldCollection.cs]: missing comment?

File: EPPlus/Table/PivotTable/ExcelPivotTableFieldCollection.cs [hunk 5]
```diff
@@ -36,235 +36,304 @@
...
-		/// <param name="name"></param>
-		/// <returns></returns>
+		internal void Clear()
+		{
+			myList.Clear();
+		}
+		#endregion
+	}
+
+	public class ExcelPivotTableFieldCollection : ExcelPivotTableFieldCollectionBase<ExcelPivotTableField>
```
Comment (zpfaltersack) [inline EPPlus/Table/PivotTable/ExcelPivotTableFieldCollection.cs]: missing comment?

File: EPPlus/Table/PivotTable/ExcelPivotTableField.cs [hunk 6]
```diff
@@ -911,120 +681,216 @@ public void AddDateGrouping(eDateGroupBy groupBy, DateTime startDate, DateTime e
...
+		}
+
+		/// <summary>
+		/// Add a field to the pivot table.
+		/// </summary>
+		/// <param name="rowsNode">The row node in the xml.</param>
+		/// <param name="index">The index of the field.</param>
+		/// <param name="fieldNodeText">The text of the new node.</param>
+		/// <param name="indexAttrText">The text of the index attribute.</param>
+		/// <returns></returns>
```
Comment (EvanSchallerer) [inline EPPlus/Table/PivotTable/ExcelPivotTableField.cs]: Returns comment

File: EPPlus/Table/PivotTable/ExcelPivotTableField.cs [hunk 7]
```diff
@@ -911,120 +681,216 @@ public void AddDateGrouping(eDateGroupBy groupBy, DateTime startDate, DateTime e
...
+			return group;
+		}
+
+		/// <summary>
+		/// Set the numeric group.
+		/// </summary>
+		/// <param name="start">The start value.</param>
+		/// <param name="end">The end value.</param>
+		/// <param name="interval">The interval value.</param>
+		/// <returns></returns>
```
Comment (EvanSchallerer) [inline EPPlus/Table/PivotTable/ExcelPivotTableField.cs]: comment

**Pull request merged**